pkg/endpoint/policy.go

```diff
@@ -126,172 +99,217 @@ func (e *Endpoint) cleanUnusedRedirects(owner Owner, oldMap policy.L4PolicyMap,
 	}
 }
 
-// Must be called with global endpoint.Mutex held
-func (e *Endpoint) regenerateConsumable(owner Owner) (bool, error) {
-	c := e.Consumable
+func getLabelsMap(owner Owner, cache *policy.ConsumableCache, minID, maxID policy.NumericIdentity) (labelArrays, error) {
+	labelsMap := map[policy.NumericIdentity]labels.LabelArray{}
```

Comment:
OK. Also responding to the previous comment here, as there is no response option there:

Daemon's TriggerPolicyUpdates() incremented policy cache's iteration number, but that was not used for anything, so the BumpRevision was actually needed here. I have further refactored the TriggerPolicyUpdates to Bump the repository revision instead, so that after that your comment applies. I also changed the Opts.Apply() changedOptions callback to also bump the repo revision so that the line that I added above is no longer needed even in that case. This based on assumption that some other daemon options may require policy regeneration as well, not 100% sure of that, though.